config/config.php.dist

```diff
@@ -58,6 +58,7 @@ $config = [
     /*
      * The following settings are *filesystem paths* which define where
      * SimpleSAMLphp can find or write the following things:
+     * - 'cachedir': Where Symfony can write it's cache. 
```

Comment:
```suggestion
     * - 'cachedir': Where SimpleSAMLphp can write its cache. 
```

Reply:
My comment is more that for the user/deployer the term "Symfony" does not necessarily mean anything. It's just cache for SimpleSAML.